### PR TITLE
[OpenGL] OSX setup

### PR DESCRIPTION
**Pull request by novalain**
Setup GL context on OSX to show the triangle

### CODE REVIEW

File: src/opengl/platform/rendercontext_ogl_osx.mm [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    NSOpenGLContext* s_context;
+    NSOpenGLView* s_view;
+}
+
+namespace dw {
+
+RenderContextOSX::RenderContextOSX(const PlatformData& platformData) {
+    std::cout << "Direwolf: Setting up render context for OSX\n";
+    NSWindow* nsWindow = (NSWindow*)(platformData.windowHandle);
+    assert(nsWindow && "Failed to initialize nsWindow\n");
```
Comment (Carlbaum) [inline src/opengl/platform/rendercontext_ogl_osx.mm:21]: is there no `#include` for this?

**Review by Carlbaum — APPROVED** [review]
Looks good

File: src/opengl/platform/rendercontext_ogl_osx.mm (hunk 1 above)
Comment (novalain) [reply src/opengl/platform/rendercontext_ogl_osx.mm:21]: I thought so too, #include <cassert> seems missing

Comment (novalain) [reply src/opengl/platform/rendercontext_ogl_osx.mm:21]: But works on my machine soooooo maybe no need on OSX

**Comment by novalain** [comment]
Merging

**Pull request merged**